platform/src/search/Platform.Search/SchoolComparators/SchoolComparatorsAcademyIndexerBuilder.cs

```diff
@@ -0,0 +1,45 @@
+using Azure.Search.Documents.Indexes;
+using Azure.Search.Documents.Indexes.Models;
+using Platform.Infrastructure.Search;
+using Platform.Search.Builders;
+
+namespace Platform.Search.SchoolComparators;
+
+public class SchoolComparatorsAcademyIndexerBuilder : IndexerBuilder
+{
+    public override string Name => SearchResourceNames.Indexers.SchoolComparatorsAcademy;
+
+    public override async Task Build(SearchIndexerClient client)
+    {
+        var cosmosDbIndexer = new SearchIndexer(
+            name: Name,
+            dataSourceName: SearchResourceNames.DataSources.SchoolComparatorsAcademy,
+            targetIndexName: SearchResourceNames.Indexes.SchoolComparators)
+        {
+            FieldMappings =
+            {
+                new FieldMapping("URN") { TargetFieldName = nameof(SchoolComparatorsIndex.UKPRN) },
```

Comment:
Just checking this particular mapping is intentional

Reply:
Yeah, just a hack at the moment to get some data in the index to tweak the analyzers and lucene syntax